app/screens/channel/channel_nav_bar/channel_nav_bar.js

```diff
@@ -41,16 +39,11 @@ export default class ChannelNavBar extends PureComponent {
         switch (Platform.OS) {
         case 'android':
             height = ANDROID_TOP_PORTRAIT;
-            if (DeviceTypes.IS_TABLET) {
-                height = ANDROID_TOP_LANDSCAPE;
-            }
             break;
         case 'ios':
             height = IOS_TOP_PORTRAIT - STATUS_BAR_HEIGHT;
             if (DeviceTypes.IS_TABLET && isLandscape) {
                 height -= 1;
-            } else if (isLandscape) {
-                height = IOS_TOP_LANDSCAPE;
             }
 
             if (DeviceTypes.IS_IPHONE_X && isLandscape) {
```

Comment:
I have doubts about if this is the proper way to use same height in both formats (portrait and landscape). maybe the IPHONE_X change is no longer needed, maybe the "height -=1" is no longer needed, I don't know the details of this tricks.

Reply:
@jespino this is definitely kind of tricky:

you have the iPhone X series and the non-X ones, which you may notice that the X series have a notch and a safe area inset, the safe area insets are different when in portrait or landscape. Then there is also the iPads where for some reason on landscape the header is just 1pt less than in portrait.

Reply:
Ok, then, I think Is correct to leave it as is now. Thanks for the info :)

Reply:
Why are you removing these??? this is changing the height of the navbar and should not be the case. is this cause of the second line added to the channel title? if that is the case I highly recommend looking for an alternative.

Reply:
Yes, AFAIK, you talk with @michaelgamble about it and the final decision is to keep same height in both modes (portrait and landscape) and then add the new line in both too (when needed). Maybe I understood this wrongly and is only needed to have the same height when we going to show the extra line. @michaelgamble can you clarify that?

Reply:
not what I recall from the conversation, that conversation was to have the subtitle in the navbar, but this is not the way to accomplish that

Reply:
Now back to the previous version (no exactly the previous version because there was a bug on android that I have fixed).

Reply:
I think this is resolved.